source/simulator/stokes_matrix_free.cc

```diff
@@ -1766,6 +1787,12 @@ namespace aspect
 
         mg_smoother_A[level].estimate_eigenvalues(temp_velocity);
         mg_smoother_Schur[level].estimate_eigenvalues(temp_pressure);
+
+        if (level==0)
+          {
+            coarse_A_size = temp_velocity.size();
+            coarse_S_size = temp_pressure.size();
+          }
```

Comment:
This is unrelated, right? What is its effect?

Reply:
This is how I compute the size of the coarse vectors to print below.